scripts/ocsp-stapling.test

```diff
@@ -311,10 +311,21 @@ wolf_pid3=$!
 wait_for_readyFile $ready_file2 $wolf_pid3 $port3
 ./examples/client/client -C -A certs/ocsp/root-ca-cert.pem -W 1 -p $port3
 RESULT=$?
+[ $RESULT -ne 0 ] && printf '\n\n%s\n' "Client connection 1 failed" && exit 1
+printf '%s\n\n' "Test PASSED!"
+
+printf '%s\n\n' "------------- TEST CASE 2 SHOULD PASS ------------------------"
```

Comment:
This is the new test case (must staple, TLS 1.3) that fails without the bug fix in this commit. The rest of the changes here are just cleanup.